### PR TITLE
Add frozendict, dataclasses libraries for #11511

### PR DESCRIPTION
**Pull request by gregestren**
#11511

dataclasses is to support Python >= 3.0 and < 3.7.

### CODE REVIEW

**Comment by gregestren** [comment]
My understanding of third_party checking process is the [Bazel green team](https://bazel.build/maintaining/maintainers-guide.html#overview) must approve. Adding @meteorcloudy as reviewer accordingly.

Also adding @sdtwigg for familiarity with the purpose.

File: third_party/py/dataclasses/BUILD [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Description:
+#   dataclasses module from the Python 3.7 standard library.
+#   For backporting to Python 3.6 and below.
+
+package(default_visibility = ["//visibility:public"])
+
+licenses(["notice"])  # Python Software Foundation
+
+filegroup(
```
Comment (sdtwigg) [inline third_party/py/dataclasses/BUILD:9]: I saw the CL explicitly adding this; but why?

File: third_party/py/dataclasses/BUILD [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    name = "srcs",
+    srcs = glob(["**"]),
+    visibility = ["//visibility:public"],
+)
+
+py_library(
+    name = "dataclasses",
+    srcs = ["__init__.py"],
+    srcs_version = "PY3",  # Only works with Python 3.
```
Comment (sdtwigg) [inline third_party/py/dataclasses/BUILD:18]: Add `imports = ".",`

File: third_party/py/frozendict/BUILD [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    name = "srcs",
+    srcs = glob(["**"]),
+    visibility = ["//visibility:public"],
+)
+
+py_library(
+    name = "frozendict",
+    srcs = ["__init__.py"],
+    srcs_version = "PY2AND3",
+    visibility = ["//visibility:public"],
```
Comment (sdtwigg) [inline third_party/py/frozendict/BUILD:15]: add `imports = ".",`

File: third_party/py/dataclasses/BUILD (hunk 1 above)
Comment (gregestren) [reply third_party/py/dataclasses/BUILD:9]: Fixes failing presubmit https://buildkite.com/bazel/bazel-bazel-github-presubmit/builds/6240#65bca58b-2680-422b-aabc-4ae9cc015e5e (https://storage.googleapis.com/bazel-untrusted-buildkite-artifacts/65bca58b-2680-422b-aabc-4ae9cc015e5e/src/test/shell/bazel/srcs_test/attempt_1.log).

File: third_party/py/frozendict/BUILD (hunk 3 above)
Comment (gregestren) [reply third_party/py/frozendict/BUILD:16]: Done.

**Pull request closed without merging**